suzieq/poller/worker/coalescer_launcher.py

```diff
@@ -135,7 +137,7 @@ async def _start_coalescer(self) -> Process:
         else:
             coalescer_args_str = ''
         coalescer_args_str = f'{self.coalescer_bin} {coalescer_args_str}'
-        coalescer_args = coalescer_args_str.strip().split()
+        coalescer_args = shlex.split(coalescer_args_str.strip())
```

Comment:
If the problem is building the list from the string, wouldn't be better to directly create a list? So having something like this:
```python
coalescer_args = [self.coalescer_bin]
if self.config_file:
   coalescer_args += ['-c', self.config_file]
```

Reply:
Sure, that'd work too. This seems simpler. The user in this case had a dir with weird characters in it such as (, \, space etc.